src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -121,6 +121,11 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// </summary>
     public string? NetTracePath { get; set; }
 
+    /// <summary>
+    /// Path to assemblies to reference when invoking dotnet-pgo to process a .nettrace collected from a separate device. Necessary for mobile platforms.
+    /// </summary>
+    public string? ReferenceAssembliesDir { get; set; }
```

Comment:
Done! I wanted to specify that this property is a directory that contains all the assemblies that are referenced in the .nettrace collected, and instead of having the user pass `$(PublishDir)*.dll` for example, having them pass in the dir.

But as you mention, this is a bit limiting, as the intended argument allows for multiple `--reference`. Perhaps this could be an ItemGroup passed to MonoAOTCompiler?

Reply:
Yeah, `ITaskItem[]` makes sense.

Reply:
Done!